docs/source/user-guide/guide_indexes.rst

```diff
@@ -94,41 +94,41 @@ Indexes
 
     .. image:: images/indexes/edit-index-fields.png
       :alt: A portion of the data dictionary creation admin page with a focus on editable form that appears after you click the gear icon to edit a specific index field.
 
-6. You can enter additional index fields by clicking "Add field" (in this index section) and repeating steps 2-3 in this Index Fields section again.
-7. After defining the index fields, click "Submit Index" to save the index temporarily. Remember that the Data Dictionary and it's respective fields and indexes are not completely saved until the form is submitted by clicking "Save".
+6. You can enter additional index fields by clicking "Add field to index" (in this index section) and repeating steps 2-3 in this Index Fields section again.
+7. After defining the index fields, click "Save index" to save the index temporarily. Remember that the Data Dictionary and it's respective fields and indexes are not completely saved until the form is submitted by clicking "Save".
    
   .. image:: images/indexes/edit-index-box.png
-    :alt: A portion of the data dictionary creation admin page with a focus on the indexes box, specifically meant to reference the "Submit Index" button.
+    :alt: A portion of the data dictionary creation admin page with a focus on the indexes box, specifically meant to reference the "Save index" button.
 
 8. Note that after submitting, an informational table will appear listing the indexes you have created thus far and their names, types, and fields (field name and field length).
 
   .. image:: images/indexes/add-index-informational-table.png
     :alt: A portion of the data dictionary creation admin page with a focus on informational table that appears after saving an index.
 
  a. You also have the ability to edit or modify the indexes you've created so far by clicking the gear icon on the right-hand side of this table.
- b. Clicking the gear icon will open an editable form that will show the previously entered information for the respective index. You can then modify this information and either: "Save" (retain the modifications), "Cancel Index" (close the editable form box without retaining any modifications) or "Delete Index" (remove the respective index entirely).
+ b. Clicking the gear icon will open an editable form that will show the previously entered information for the respective index. You can then modify this information and either: "Save index edit" (retain the modifications), "Cancel index edit" (close the editable form box without retaining any modifications) or "Remove index" (remove the respective index entirely).
 
     .. image:: images/indexes/edit-index.png
       :alt: A portion of the data dictionary creation admin page with a focus on the edit index box.
```

Comment:
line 136, small tweak to the first sentence:
Note that once you have saved an index, it is not possible to edit the fields within the index at this time. ...